test/javascripts/unit/components/query-result-test.js.es6

```diff
@@ -0,0 +1,103 @@
+import { moduleFor } from "ember-qunit";
+import { test } from "qunit";
+
+moduleFor("component:query-result");
+
+test("it transforms data for a graph", function (assert) {
+  const results = {
+    colrender: [],
+    result_count: 2,
+    columns: ["user", "like_count"],
+    rows: [
+      ["user1", 10],
+      ["user2", 20],
+    ],
+  };
+  this.subject().setProperties({
+    content: results,
+  });
+
+  assert.deepEqual(
+    this.subject().get("graphLabels"),
```

Comment:
Won't note here everywhere but get is probably not needed each time you use it, if subject is null get won't save you